opnode/test/system_test.go

```diff
@@ -100,8 +100,8 @@ func defaultSystemConfig(t *testing.T) SystemConfig {
 			},
 		},
 		Loggers: map[string]log.Logger{
-			"verifier":  testlog.Logger(t, log.LvlError),
-			"sequencer": testlog.Logger(t, log.LvlError),
+			"verifier":  testlog.Logger(t, log.LvlInfo),
+			"sequencer": testlog.Logger(t, log.LvlInfo),
```

Comment:
I always end up changing these log levels to get info for my tests. Can we just default them to info level?

Reply:
won't change it for now, we can revisit logging separately